python/pyramid/audit/csrf-origin-check-disabled.yaml

```diff
@@ -0,0 +1,32 @@
+rules:
+  - id: pyramid-csrf-origin-check-disabled
+    message: >-
+      Origin check for the CSRF token is disabled for this view.
+      This might represent a security risk if the CSRF storage policy is not known to be secure.
```

Comment:
Could you include another sentence that talks about the fix, similar to [here](https://github.com/returntocorp/semgrep-rules/blob/f0dfbdd231f6218aca034101b86fa3497f06890f/python/pyramid/audit/csrf-origin-check-disabled-globally.yaml#L13)?